_docs/2017-01-01-index.md

```diff
@@ -170,6 +169,6 @@ outcomes. Accompanied with side chains it will make possible to accomplish
 such tasks as provably fair distribution of winnings in games. The
 application of Control Layer lies well beyond gaming and gambling. Identity
 management, credit system and more will be a part of Cardano Platform.
-We are also aiming to evolve Cardano SL wallet program called [Daedalus]()
+We are also aiming to evolve Cardano SL wallet program called Daedalus
```

Comment:
Maybe `Cardano SL wallet application` would be better here. Calling it "app" is reducing it to a marketing trick, but "program" doesn't fit as well as it could here.